devicetypes/smartthings/zwave-metering-switch.src/zwave-metering-switch.groovy

```diff
@@ -199,6 +199,16 @@ def zwaveEvent(physicalgraph.zwave.commands.manufacturerspecificv2.ManufacturerS
 	result << createEvent(descriptionText: "$device.displayName MSR: $msr", isStateChange: false)
 }
 
+def zwaveEvent(physicalgraph.zwave.commands.notificationv3.NotificationReport cmd){
+	if ((cmd.notificationType == 0x08) && zwaveInfo?.mfr?.equals("018C")) {
+		if (cmd.event == 0x02) {
```

Comment:
Looking at the Notification Type of Power Management (0x08) and the following state/events:

0x02: AC Mains Disconnected
0x03: AC Mains re-connected

I think this makes sense. 

The only think I would ask is that, does it makes sense to have this  for all devices that have "018C" manufacturer or just do it for this particular device.

Reply:
This Korean company is just started to work with SmartThings. Currently requested devices that have a switch seem to have the same approach. So I can say having 018C mfr in the code makes sense currently.